datasets/ml_datasets/dataset.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dataset:
+  # The `dataset` block includes properties for your dataset that will be shown
+  # to users of your data on the Google Cloud website.
+
+  # Must be exactly the same name as the folder name your dataset.yaml is in.
+  name: ml_datasets
+
+  # A friendly, human-readable name of the dataset
+  friendly_name: ~
+
+  # A short, descriptive summary of the dataset.
+  description: ~
+
+  # A list of sources the dataset is derived from, using the YAML list syntax.
+  dataset_sources: ~
+
+  # A list of terms and conditions that users of the dataset should agree on,
+  # using the YAML list syntax.
+  terms_of_use: ~
+
+resources:
+  # A list of Google Cloud resources needed by your dataset. In principle, all
+  # pipelines under a dataset should be able to share these resources.
+  #
+  # The currently supported resources are shown below. Use only the resources
+  # you need, and delete the rest as needed by your pipeline.
+  #
+  # We will keep adding to the list below to support more Google Cloud resources
+  # over time. If a resource you need isn't supported, please file an issue on
+  # the repository.
+
+  - type: bigquery_dataset
+    # Google BigQuery dataset to namespace all tables managed by this folder
+    #
+    # Required Properties:
+    #   dataset_id
+    #
+    # Optional Properties:
+    #   friendly_name (A user-friendly name of the dataset)
+    #   description   (A user-friendly description of the dataset)
+    #   location      (The geographic location where the dataset should reside)
+    dataset_id: ml_datasets
```

Comment:
"datasets" is a bit redundant. Can we call this `ml_samples` to be clear this isn't intended for production?

Reply:
This is an existing dataset that I'm adding to.

Reply:
@ivanmkc The worry is that `ml_datasets` covers too much ground. It can mean datasets for tutorials, for actual research, and so on.

@tswast `ml_samples` is good. Or perhaps make it even more specific like `ai_platform_tutorials` or `ml_tutorials`?

Reply:
Good to know. I see there are already tables here: https://pantheon.corp.google.com/bigquery?d=ml_datasets&p=bigquery-public-data&page=dataset

I'm okay with adding to the existing dataset. It'd be too much to ask to move, and I don't want 2 for ml sample data.